packages/amplify-graphql-api-construct/src/amplify-graphql-api.ts

```diff
@@ -126,6 +126,11 @@ export class AmplifyGraphqlApi extends Construct {
    */
   private readonly conflictResolution: ConflictResolution | undefined;
 
+  /**
+   * Be very careful editing this value. This is the string that is used to identify graphql stacks in BI metrics
+   */
+  private readonly stackType = 'api-AppSync';
```

Comment:
Can you confirm that a change in this value will cause [this expect](https://github.com/aws-amplify/amplify-category-api/pull/2039/files#diff-08d9ca28779728e317561c46c3538f4d2b813f4ccadaeb8efa5ae5cbed49d276R189) to fail?

Reply:
Confirmed changing stackType to `al-rulez` did cause failure.

```
  ● storeOutput › BI metrics output › stores expected BI metadata in stack description

    expect(received).toMatchObject(expected)

    - Expected  - 1
    + Received  + 1

      Object {
        "createdWith": "1.3.0",
    -   "stackType": "api-AppSync",
    +   "stackType": "al-rulez",
      }
```